app/scripts/ui/palette.js

```diff
@@ -10,6 +10,9 @@ function Palette(options, ui) {
     .addClass('dt-palette')
     .addClass(options.vertical ? 'dt-vertical' : 'dt-horizontal');
 
+  this.topOffset = options.hasOwnProperty('topOffset') ? options.topOffset : -2;
+  this.leftOffset = options.hasOwnProperty('leftOffset') ? options.leftOffset : 1;
```

Comment:
Where is the offset used in the end and why -2 / 1? When the stamp replaces the default stamp icon in the menu? I guess some comments could be useful if this needs to be adjusted when we add new stamps.

Reply:
I will clean this up also - I was tweaking the placement line up the menu borders to match the new UI spec.